modules/stochastic_tools/src/samplers/Cartesian1DSampler.C

```diff
@@ -0,0 +1,77 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "Cartesian1DSampler.h"
+
+registerMooseObjectAliased("StochasticToolsApp", Cartesian1DSampler, "Cartesian1D");
+
+InputParameters
+Cartesian1DSampler::validParams()
+{
+  InputParameters params = Sampler::validParams();
+  params.addClassDescription("Provides complete Cartesian product for the supplied variables.");
+  params.addRequiredParam<std::vector<Real>>(
```

Comment:
consider to use std::vector< std::vector< 
it would not be fun to check a 30-length vector with no separators

what do the other objects do in StochasticTools

Reply:
This is exactly the same syntax as `CartesianProduct`, it would probably be best to keep them consistent.